pallets/collator-selection/README.md

```diff
@@ -1 +1,4 @@
-License: Unlicense
\ No newline at end of file
+License: Apache-2.0
+
+
+* To initiate rewards on chains that will get low fees, an ED needs to be sent to the pot address.
```

Comment:
???

Reply:
Basically the fees will probably never add up to greater than the ED, so the staking pot would never get created on-chain. So someone needs to donate the ED to the staking pot, and then the ED serves as the base value of the account.

Reply:
I was more concerned by why this is in the README? :D

Reply:
Yeah, should be in the pallet docs.

Reply:
ya, will move, I wasn't exactly sure where to put it